packages/client-web-theme/src/index.css

```
@@ -0,0 +1,299 @@
+ *                                as templates migrate to <VCButton> /
+ *                                <VCAlert> / etc.
+ */
+
+@import "@authup/client-web-kit-theme";
+
+/*
+ * Scan this package's own src so class strings declared inside
+ * `clientWebTheme()` (none today, but a hook for future overrides)
+ * land in the JIT corpus.
+ */
+@source ".";
```

Stylelint (17.11.1)

[error] Unexpected unknown at-rule "`@source`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)